Week1-

AnujaBujurge29 opened the pull request.

ethanarns commented on 01-node-tutorial/answers/prompter.js:
```diff
@@ -1,24 +1,24 @@
-    callback(resultHash);
-  });
-};
+// const getBody = (req, callback) => {
```
I think you may have forgotten to do something here. Commenting this out unfortunately makes it break and not compile.

ethanarns commented on 01-node-tutorial/answers/11-fs-async.js:
```diff
@@ -0,0 +1,26 @@
+    }
+    // console.log(result);
+    const first = result;
+    readFile('../content/second.txt', 'utf8', (err, result) => {
```
Good work! You got it!

ethanarns reviewed (commented):
Unfortunately, I think you forgot to edit prompter.js here. It does not compile because a big portion of it was removed via commenting. However, the other fixes are good!

AnujaBujurge29 commented:
Hi,
Thanks for your review. Please find new Pull request:

https://github.com/AnujaBujurge29/node-express-course/pull/3

Let me know if we still have errors.

Thank you for your guidance.
Anuja




[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03/15/24,
09:38:06 AM

On Thu, Mar 14, 2024 at 11:48 PM Ethan Messinger-Arns <
***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Unfortunately, I think you forgot to edit prompter.js here. It does not
> compile because a big portion of it was removed via commenting. However,
> the other fixes are good!
> ------------------------------
>
> In 01-node-tutorial/answers/prompter.js
> <https://github.com/AnujaBujurge29/node-express-course/pull/1#discussion_r1525754823>
> :
>
> > -  req.on("data", function (data) {
> -    body += decode.write(data);
> -  });
> -  req.on("end", function () {
> -    body += decode.end();
> -    const body1 = decodeURI(body);
> -    const bodyArray = body1.split("&");
> -    const resultHash = {};
> -    bodyArray.forEach((part) => {
> -      const partArray = part.split("=");
> -      resultHash[partArray[0]] = partArray[1];
> -    });
> -    callback(resultHash);
> -  });
> -};
> +// const getBody = (req, callback) => {
>
> I think you may have forgotten to do something here. Commenting this out
> unfortunately makes it break and not compile.
> ------------------------------
>
> In 01-node-tutorial/answers/11-fs-async.js
> <https://github.com/AnujaBujurge29/node-express-course/pull/1#discussion_r1525754979>
> :
>
> > @@ -0,0 +1,26 @@
> +const { readFile, writeFile } = require('fs')
> +
> +readFile('../content/first.txt', 'utf8', (err, result) => {
> +    if (err) {
> +        console.log(err);
> +        return
> +    }
> +    // console.log(result);
> +    const first = result;
> +    readFile('../content/second.txt', 'utf8', (err, result) => {
>
> Good work! You got it!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AnujaBujurge29/node-express-course/pull/1#pullrequestreview-1938111568>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6BNFZIHJOETFSQC2PTCSOLYYJ4TLAVCNFSM6AAAAABEMEIQAOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTSMZYGEYTCNJWHA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>
